core/pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>commons-beanutils</groupId>
       <artifactId>commons-beanutils</artifactId>
-      <version>1.7.0</version>
+      <version>1.9.3</version>
```

Comment:
Too new, and see #134.

Reply:
Can this be revisited? We are running into some serious dependency issues because its at 1.8.3